src/views/components/dashboard/charts/chart-edit.vue

```diff
@@ -237,6 +237,16 @@ limitations under the License. -->
           @change="setItemConfig({ type: 'height', value: $event.target.value })"
         />
       </div>
+      <div class="flex-h mb-5" v-show="!isChartType">
+        <div class="title grey sm">{{ $t('maxItemNum') }}:</div>
+        <input
+          type="number"
+          min="1"
+          class="rk-chart-edit-input long"
+          :value="itemConfig.maxItemNum"
```

Comment:
Please set the default value of `maxItemNum` to `10`.

Reply:
I feel that all the default data is configured on the back end through the 'queryGetAllTemplates' API, There may be a problem with setting the default value at the front end. 
![image](https://user-images.githubusercontent.com/5698982/116074411-4c5e9580-a6c4-11eb-928c-9a91476f64c7.png)

Reply:
The default works when this config missing. Which are really missing for all configuration.

Reply:
ok, It makes sense